extensions/dicom-microscopy/src/utils/loadAnnotation.js

```diff
@@ -0,0 +1,70 @@
+import getDicomWebClient from './dicomWebClient';
+
+export default function loadAnnotation({
+  microscopyService,
+  displaySet,
+  extensionManager,
+  servicesManager,
+}) {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const { metadata } = displaySet;
+
+      const dicomMicroscopyModule = await microscopyService.importDicomMicroscopyViewer();
+
+      const client = getDicomWebClient({
+        extensionManager,
+        servicesManager,
+      });
+
+      const viewportId = servicesManager.services.viewportGridService.getActiveViewportId();
+      const managedViewers = microscopyService.getManagedViewersForViewport(viewportId);
+      const managedViewer = managedViewers[0];
+
+      client
+        .retrieveSeriesMetadata({
+          studyInstanceUID: metadata.StudyInstanceUID,
+          seriesInstanceUID: metadata.SeriesInstanceUID,
+        })
+        .then(retrievedMetadata => {
+          const annotations = retrievedMetadata.map(metadata => {
```

Comment:
A simple map function is easier when you just do a single transform, eg:
metadata => new dicomMicroscopyModules.metadata.Micro...({ metadata })